clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsCramTests.scala

```diff
@@ -856,6 +863,54 @@ trait WgsCramTests { self: BaseIntegrationSpec =>
     }
   }
 
+  it should "automatically set regulatory designation to RESEARCH_ONLY for crams" in {
+    val project = s"project$randomId"
+    val sample = s"sample$randomId"
+    val version = 3
+
+    val md5Contents = randomId
+
+    val cramName = s"$randomId.cram"
+    val craiName = s"$cramName.crai"
+
+    val rootSource =
+      rootTestStorageDir.resolve(s"cram/$project/$sample/v$version/")
+    val cramSource = rootSource.resolve(cramName)
+    val craiSource = rootSource.resolve(craiName)
+
+    val key = TransferWgsCramV1Key(Location.GCP, project, sample, version)
+    val metadata = TransferWgsCramV1Metadata(
+      cramPath = Some(cramSource.toUri),
+      craiPath = Some(craiSource.toUri),
+      cramMd5 = Some(Symbol(md5Contents)),
+      regulatoryDesignation = None
```

Comment:
can't you omit this line?

Reply:
I'm explicitly setting it to None to make it clear in the test that the value should be changed by the server.